build-osx.sh

```diff
@@ -248,8 +248,11 @@ case $command in
     --uninstall-surge)
         run_uninstall_surge
         ;;
-    *)
+    "")
```

Comment:
Is this actually spdlog-related. I honestly can't tell.

Reply:
Good eyes, and very sorry it is not; that got swept into this as I fixed that bug (which was basically you could use build-osx --anything and it worked) while developing spdlog.

If you want, I can pull it out and put it in a separate PR.

Reply:
OK I removed this and the .gitignore from this PR. I will squash all the commits when done with any other comments and before I merge. I'll do a separate PR for the .DS_Store and the "" vs * in the build.

Reply:
OK I pushed these two diffs into PR #153 rather than this one. Thanks!